prod.Dockerfile

```diff
@@ -1,21 +1,19 @@
-FROM node:18-alpine as builder
+FROM oven/bun:alpine as builder
 
 WORKDIR /app
 
-COPY package.json yarn.lock ./
-RUN yarn install --frozen-lockfile
+COPY package.json bun.lockb ./
+RUN bun install --frozen-lockfile
 COPY . .
-RUN yarn build:ts
 
-FROM node:slim
+FROM oven/bun:alpine
 
 ENV NODE_ENV production
 USER node
 
 WORKDIR /app
-COPY --from=builder /app/package.json /app/yarn.lock ./
-RUN yarn install --frozen-lockfile --production
+COPY --from=builder /app/package.json /app/bun.lockb ./
+RUN bun install --frozen-lockfile --production
 COPY --from=builder /app/dist ./dist
 
-CMD ["yarn", "fastify", "start", "-a", "::", "-l", "info", "dist/app.js"]
-
+CMD ["bun", "fastify", "start", "-a", "::", "-l", "info", "dist/app.js"]
```

Comment:
If you try running this command (without Docker), you're likely to find this won't work.
- We're no longer transpiling ts -> js
- Once you replace this app.js with app.ts, I expect you'll find that the `fastify` CLI does not support TypeScript files.

Possible solution: remove dependence on Fastify CLI by using `fastify eject`, then try and see if it works out of the box, if not we might need some more adjustments.

Reply:
Can we use `bun build:ts` first then let the command work normally, cause then it will be able to find `dist/app.js`?

Yes the fastify CLi does not support typescript files, by using `fastify eject-ts` there is a new `server.ts` file generated to let it function as a standalone app, Im figuring it out still.

Will update soon